herd/libdir/aarch64deps.cat

```diff
@@ -7,11 +7,11 @@
  * jade.alglave@arm.com, referring to version number:
  * 814a6fc1610ec1a24f2cbd178e171966375626ac
  * For a textual version of the model, see section B2.3 of the Armv8 ARM:
- *   https://developer.arm.com/docs/ddi0487/latest/arm-architecture-reference-manual-armv8-for-armv8-a-architecture-profile
+ *   https://developer.arm.com/docs/dtri0487/latest/arm-architecture-reference-manual-armv8-for-armv8-a-architecture-profile
```

Comment:
Not sure if this change is intentional but the new link is not working. I also tried the old one which doesn't really link to the latest version any more but it get redirected. Otherwise we can use:

https://developer.arm.com/documentation/ddi0487/ha/